solutions/standard/catalogValidationValues.json.template

```diff
@@ -3,6 +3,5 @@
   "resource_group_name": $PREFIX,
   "tags": $TAGS,
   "existing_kms_instance_crn": "crn:v1:bluemix:public:hs-crypto:us-south:a/abac0df06b644a9cabc6e44f55b3880e:e6dce284-e80f-46e1-a3c1-830f7adff7a9::",
```

Comment:
oh, this should actually be changed to:
`"existing_kms_instance_crn": $HPCS_US_SOUTH_CRN`

Reply:
done